MCprep_addon/conf.py

```diff
@@ -36,10 +45,10 @@ class MCprepEnv:
 	def __init__(self):
 		self.data = None
 		self.json_data = None
-		self.json_path: Path = Path(os.path.dirname(__file__), "MCprep_resources", "mcprep_data.json")
-		self.json_path_update: Path = Path(os.path.dirname(__file__), "MCprep_resources", "mcprep_data_update.json")
+		self.json_path: PathLike = Path(os.path.dirname(__file__), "MCprep_resources", "mcprep_data.json")
```

Comment:
Can you share more of the rational and reason for changing to pathLike for typing? Just curious what problems it solves (in some cases I understand making typing more general, but we _do_ know it should be a Path class instance)

Reply:
That was me being indecisive. I made it like that for later change. So change back to Path.